tests/query/transactions/test_query_transactions.py

```diff
@@ -0,0 +1,29 @@
+import allure
+import iroha
+
+from tests import client,account_id
+from tests.helpers import generate_public_key
+
+
+@allure.label("sdk_test_id", "query_all_transactions")
+def test_query_all_transactions(GIVEN_registered_transaction):
+    with allure.step('WHEN client queries all transactions'):
+        all_txs = client.query_all_transactions()
+    with allure.step('THEN there should be some transactions present'):
+        assert len(all_txs) > 0, "No transactions found in the system"
+
+
+@allure.label("sdk_test_id", "query_all_transactions_by_account")
+def test_query_all_transactions_by_account(
+        GIVEN_registered_transaction):
+    with allure.step('WHEN client has performed transactions'):
+        all_txs = client.query_all_transactions_by_account(account_id)
+    with allure.step('THEN there should be some transactions present'):
+        assert len(all_txs) > 0, "No transactions found in the system"
+
+@allure.label("sdk_test_id", "test_query_transaction_by_hash")
+def test_query_transaction_by_hash(
+        GIVEN_registered_transaction):
+    with allure.step(
+            f'WHEN client queries for transaction by hash THEN there should be transaction present"'):
+        tx = client.query_transaction_by_hash(bytes.fromhex(GIVEN_registered_transaction))
```

Comment:
I don't see any assert of this test. Please write with `allure.step('THEN there should be transaction present')` .